### PR TITLE
[NO TESTS NEEDED] Makefile: split install.docker-docs from install.docker

### PR DESCRIPTION
**Pull request by liuming50**
This allows end users to be able to choose to not install any docs.

Signed-off-by: Ming Liu <liu.ming50@gmail.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by liuming50** [comment]
Changes in patch set V2:

1 Added one more fix to Makefile.

**Comment by mheon** [comment]
Test failures during RPM build phase look legitimate. @baude Is there a make target that invokes?

**Comment by baude** [comment]
it looks something about the systemd files is the problem and there is a modification there in the PR.

File: Makefile [hunk 1]
```diff
@@ -543,14 +543,18 @@ install.cni:
...
 	install ${SELINUXOPT} -m 755 -d ${DESTDIR}${SYSTEMDDIR}  ${DESTDIR}${USERSYSTEMDDIR} ${DESTDIR}${TMPFILESDIR}
 	install ${SELINUXOPT} -m 644 contrib/systemd/system/podman-docker.conf -t ${DESTDIR}${TMPFILESDIR}
 
+.PHONY: install.docker-docs
+install.docker-docs: docker-docs
+	install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
+	install ${SELINUXOPT} -m 644 docs/build/man/docker*.1 -t $(DESTDIR)$(MANDIR)/man1
+
 .PHONY: install.systemd
+ifeq (,$(findstring systemd,$(BUILDTAGS)))
```
Comment (edsantiago) [inline Makefile:557]: shouldn't this be `ifneq`?

Comment (liuming50) [reply Makefile:557]: @edigaryev Arha, you are absolutely right, my bad, will make a V3.

**Comment by liuming50** [comment]
Changes in patch set V3:

1 Fixed a typo as pointed out by @edigaryev

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
Thanks @liuming50

**Pull request merged**